indexer/editable_map_object.hpp

```diff
@@ -46,6 +46,8 @@ struct LocalizedName
 class EditableMapObject : public MapObject
 {
 public:
+  using LocalizedStreet = pair<string /* default */, string /* localized */>;
```

Comment:
Вообще - нужна )

> On Apr 8, 2016, at 15:54, Dmitry Yunitsky <notifications@github.com> wrote:
> 
> In indexer/editable_map_object.hpp <https://github.com/mapsme/omim/pull/2850#discussion_r59019931>:
> 
> > @@ -46,6 +46,8 @@ struct LocalizedName
> >  class EditableMapObject : public MapObject
> >  {
> >  public:
> > +  using LocalizedStreet = pair<string /* default */, string /* localized */>;
> @deathbaba <https://github.com/deathbaba> подскажи
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2850/files/a95dab907b1c262067b85115495a233eb4f185d6#r59019931>